oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/common/components/EventSubmitForm.jsx

```diff
@@ -34,7 +34,7 @@ export const EventSubmitForm = ({
   const queryClient = useQueryClient();
 
   const { mutate, isError, isLoading, reset } = useMutation(
-    () => http.post(request.links?.comments, values),
+    () => http.post(request.links?.comments + "?expand=1", values),
```

Comment:
Missing payload content sanitization (use requests form cfg api to get allowed html tags)

Reply:
@mirekys It is already sanitized in https://github.com/oarepo/oarepo-requests/blob/3a57df355133d5d794d8f75fe97ee50a42debe15/oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/common/components/RequestCommentInput.jsx#L9 before being put into state.

Reply:
@mirekys It is sanitized in the input itself before being placed into state https://github.com/oarepo/oarepo-requests/blob/3a57df355133d5d794d8f75fe97ee50a42debe15/oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/common/components/RequestCommentInput.jsx#L9